zeppelin-distribution/src/bin_license/LICENSE

```diff
@@ -188,6 +188,7 @@ The following components are provided under the CDDL License.
 
     (CDDL 1.0) javax.activation (javax.activation:activation:jar:1.1.1 - http://java.sun.com/javase/technologies/desktop/javabeans/jaf/index.jsp)
     (CDDL 1.1) Jersey (com.sun.jersey:jersey:jar:1.9 - https://jersey.java.net/)
+    (CDDL 1.1) (GPL2) jersey-core (org.glassfish.jersey.core:jersey-core:2.22.2 - https://jersey.java.net/)
```

Comment:
I think we can remove (GPL2) here, if we're providing jersey-core under CDDL 1.1 license.
Also all transitive dependency (e.g. org.glassfish.hk2:hk2-api, etc) need to be addressed, if license of jersey-core doesn't cover them.